### DISCUSSION TITLE
xui cert problem

### DISCUSSION THREAD

**Discussion by linchan6369** [FranzKafkaYu/x-ui#232] [Q&A]
hi 
 i instal xui  and i can see cert and key file at root 
but when i put this directory on xui pannel and try to save  see this error everytime 

Failed to modify settings: cert file </root/cert.crt > or key file </root/private.key> invalid: open /root/cert.crt : no such file or directory

**Comment by LeoBonharts** [comment]
Hi, for solving this problem set panel root path to  ./   Then save and restart it will detect certs in root patch.

**Comment by mahdikhorsand** [comment]
change it to /root/cert/yourdomain.cer
and /root/cert/yourdomain.key
replace your domain in above text and paste it to x-ui panel
